### DISCUSSION TITLE
Service charge and tip

### DISCUSSION THREAD

**Discussion by ahorasoyfelipe** [Blair2004/NexoPOS#342] [Gastro 4.x Ideas & Issues]
It is necessary for restaurants to be able to have the tip option that goes on the account, as the tax is, it is the same functionality but carried a service charge and tip, which can be chosen if it is by amount or percentage and what can be editable in the order itself

That this field can be editable to change the amount if the customer leaves a different tip
![Captura de Pantalla 2021-10-17 a la(s) 22 55 56](https://user-images.githubusercontent.com/80654041/137657842-6dfe8085-bb09-41dc-8859-c0a829f61ae1.png)

It is to use the same logic of the tax, since it complies with everything, it is automatically shown on the receipt and at the same time a different control is carried out on the sale, since the tip is paid to the waiter and is not part of the sale itself, and keep the separate control as well as the sale of the tax, it would be ideal

Please consider this as it is essential to be able to use the system in restaurants.

**Comment by Blair2004** [comment]
So the tip there work as a tax 🤔🤔🤔

**Reply by ahorasoyfelipe** [reply]
Yes, in terms of the way of working it works like the tax, with the only difference is that the amount needs to be modifiable before charging, so if the client decides not to leave a tip, I can put 0, or if I leave an amount Less than suggested I can put it and so I can know exactly how much tip was charged in a shift and then pay that to the waiters

**Reply by aandrade** [reply]
This feature is very necessary

**Comment by Blair2004** [comment]
Thank you for bringing this back to our attention. We need to consider the reports, how this will affect the actual reports then  if the outcome is without any confusion, we'll hold the feature for NexoPOS 4.7.x
